apps/extension/src/utils/download-proving-keys.ts

```diff
@@ -16,6 +16,13 @@ const main = () => {
   const downloads = Object.values(provingKeys).map(async name => {
     const file = `${name}_pk.bin`;
     const outputPath = path.join(binDir, file);
+
+    // Check if the file already exists
+    if (fs.existsSync(outputPath)) {
+      console.log(`${file} already downloaded.`);
+      return;
+    }
```

Comment:
if we're going to do this in the script, we should have a cli flag to disable this check.

Reply:
turbo should be able to correctly understand when to cache key download, and thus handle this condition transparently, if we configure it correctly. the ci uses it successfully to restore downloaded keys.